site/content/docs/0.0/migration-from-boosted.md

```diff
@@ -11,6 +11,25 @@ toc: true
 
 This guide provides information on how to migrate from Boosted (latest release) to OUDS Web.
 
+Let's start with the most obvious change you'll notice: the name. **Boosted is now OUDS Web.**
+
+- The NPM package is now `@ouds/web`.
+- The CSS and JS files have been renamed:
```

Comment:
Do we mention the removal of `boosted-helvetica.*` files?

Reply:
Ah mb, I was looking for boosted-helvetica, but yeah your right it's litterally right below. Closing it for sure, you can merge it.